com.unity.netcode.gameobjects/Editor/CodeGen/CodeGenHelpers.cs

```diff
@@ -6,39 +6,140 @@
 using Mono.Cecil;
 using Mono.Cecil.Cil;
 using Mono.Cecil.Rocks;
-using Unity.Collections;
 using Unity.CompilationPipeline.Common.Diagnostics;
 using Unity.CompilationPipeline.Common.ILPostProcessing;
-using UnityEngine;
+using Unity.Netcode.Shared;
 
 namespace Unity.Netcode.Editor.CodeGen
 {
+    namespace Import
+    {
+#pragma warning disable IDE1006 // disable naming rule violation check
+        internal static class ModuleNames
+        {
+            public const string NetcodeRuntime = "Unity.Netcode.Runtime.dll";
+            public const string NetStandard = "netstandard.dll";
+            public const string UnityEngineCoreModule = "UnityEngine.CoreModule.dll";
+        }
+
+        internal static class TypeNames
+        {
+            public const string NetworkManager = "Unity.Netcode.NetworkManager";
+            public const string NetworkManager_RpcReceiveHandler = "Unity.Netcode.NetworkManager/RpcReceiveHandler";
+            public const string NetworkBehaviour = "Unity.Netcode.NetworkBehaviour";
+            public const string NetworkBehaviour___RpcExecStage = "Unity.Netcode.NetworkBehaviour/__RpcExecStage";
```

Comment:
this could be the bug we're looking for (ILPP failing to make RpcExecStage protected from internal)

Reply:
How does adding a few string prevents the change from `protected` to `internal` ? (Not arguing it doesn't, just really curious how one leads to the other)

Reply:
more specifically, this part: `viour/__RpcExecSt`